chef/cookbooks/neutron/recipes/server.rb

```diff
@@ -224,7 +224,8 @@
       vxlan_end: vni_end,
       vxlan_mcast_group: node[:neutron][:vxlan][:multicast_group],
       external_networks: physnets,
-      mtu_value: mtu_value
+      mtu_value: mtu_value,
+      l2pop_settings: node[:neutron][:l2pop]
```

Comment:
I prefer not to pass down the whole hash, just the value. This way we are exposing the internal structure of `l2pop` for the template.